libmamba/src/api/configuration.cpp

```diff
@@ -629,13 +629,21 @@ namespace mamba
             }
         }
 
+        bool is_root_prefix(const fs::u8path& prefix)
```

Comment:
Note to reviewers: I am thinking of changing it to use `validate_root_prefix` instead.

Reply:
I would assume a `validate_root_prefix` function would use `is_root_prefix` and do something special if it returns false. It's often better to separate the predicate from the reaction.

Reply:
Yes it does, but `validate_root_prefix` also performs more/comprehensive checks.

Reply:
I meant replacing the use of this function by `validate_root_prefix`.

Reply:
Ah yes, at usage point, ok then 👍🏼